From e33b82889625b6a72e0be91c5023a30d53136a80 Mon Sep 17 00:00:00 2001
From: Friedemann Kleint <Friedemann.Kleint@qt.io>
Date: Mon, 21 Aug 2023 08:24:28 +0200
Subject: [PATCH 6/6] Fix translation context of
 QBluetoothUuid::WeightScaleFeature

Amends 439e818f7843b6a117e1bfe755e313022e14aafa.

Pick-to: 6.6 6.5 6.2 5.15
Change-Id: If08fc84a495d493b463cc71de43237b1cc950a5b
Reviewed-by: Alex Blasche <alexander.blasche@qt.io>
(cherry picked from commit a6bac31e41b71aeff6272cf8c34f02b237b21184)
---
 src/bluetooth/qbluetoothuuid.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/bluetooth/qbluetoothuuid.cpp b/src/bluetooth/qbluetoothuuid.cpp
index e0ad090a..517465bd 100644
--- a/src/bluetooth/qbluetoothuuid.cpp
+++ b/src/bluetooth/qbluetoothuuid.cpp
@@ -1078,7 +1078,7 @@ QString QBluetoothUuid::characteristicToString(CharacteristicType uuid)
     case QBluetoothUuid::BodyCompositionMeasurement: return QBluetoothServiceDiscoveryAgent::tr("Body Composition Measurement");
     case QBluetoothUuid::WeightMeasurement: return QBluetoothServiceDiscoveryAgent::tr("Weight Measurement");
     case QBluetoothUuid::CharacteristicType::WeightScaleFeature:
-        return QBluetoothDeviceDiscoveryAgent::tr("Weight Scale Feature");
+        return QBluetoothServiceDiscoveryAgent::tr("Weight Scale Feature");
     case QBluetoothUuid::UserControlPoint: return QBluetoothServiceDiscoveryAgent::tr("User Control Point");
     case QBluetoothUuid::MagneticFluxDensity2D: return QBluetoothServiceDiscoveryAgent::tr("Magnetic Flux Density 2D");
     case QBluetoothUuid::MagneticFluxDensity3D: return QBluetoothServiceDiscoveryAgent::tr("Magnetic Flux Density 3D");
-- 
2.42.0

